WebAPI.OData.EF6/Attributes/AutomapperEnableQueryAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.OData.Query;
+using System.Linq;
+
+namespace WebAPI.OData.EF6.Attributes
+{
+    public class AutomapperEnableQueryAttribute : EnableQueryAttribute
```

Comment:
Should these attributes be named something like `AllowApplyAttribute` or `EnableApplyAttribute`?

Reply:
Depends on the goal, but for testing/example purpopes it also could be named `MyEnableQueryAttribute`. I'm open for any suggestion from your part

Reply:
Rename class to the suggested `AllowApplyAttribute`